WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -738,9 +738,6 @@ - (void)configureTableViewData
 {
     NSMutableArray *marr = [NSMutableArray array];
     
-    if (AppConfiguration.showsQuickActions && ![self isDashboardEnabled]) {
-        [marr addObject:[self quickActionsSectionViewModel]];
```

Comment:
> This PR also disables Quick Start and the onboarding prompt for the simplified UI.

Do we remove this functionality entirely, or only when the feature flags are enabled?

Reply:
Quick **Start** is removed only when the feature flags are enabled. The piece of code you're referring to has to do with Quick **Actions**. Quick Actions were actually never added since the dashboard was always enabled.

<img src="https://user-images.githubusercontent.com/25306722/209959082-68342fa8-6bf6-4287-841a-be18405fd143.png" width=40%>

For the simplified UI, we've disabled the dashboard but still don't want to add the Quick Actions card. Making this block of code redundant.

Reply:
Oh, got it! Thanks for the explanation.